test/actions/katello/organization_test.rb

```diff
@@ -144,6 +144,7 @@ class ManifestRefreshTest < TestBase
       rhel7 = katello_repositories(:rhel_7_x86_64)
       acme_org.stubs(:owner_details).returns({})
       acme_org.products.stubs(:redhat).returns([rhel7.product])
+      acme_org.stubs(:simple_content_access?).returns(false)
```

Comment:
The `finalize` method reloads the org so this stub won't work. I think you need to use Organization.any_instance here unfortunately

Reply:
Switched them all to the `Organization.any_instance` stub